my-dsl.md

```diff
@@ -1,38 +1,40 @@
 # Language
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
-
+The name of the language is django.
+[Django Project](https://www.djangoproject.com/)
```

Comment:
Interesting! I had heard of Django before, but never thought of it as a programming language. I suppose it is!